denonavr/foundation.py

```diff
@@ -521,6 +538,15 @@ def power(self) -> Optional[str]:
         """
         return self._power
 
+    @property
+    def settings_menu(self) -> Optional[str]:
+        """
+        Return the settings menu state of the device.
+
+        Possible values are: "ON" and "OFF"
```

Comment:
Please add to the comment that this function only returns a value when telnet is enabled.